### ISSUE TITLE
SFML

### ISSUE THREAD

**Issue by e-osuna-g** [genbetadev/Genbeta-Dev-Engine#1]
Tengo una duda, el SFML tenemos que agregarlo nosotros(instalarlo) o seria mejor se subiera al proyecto de git? (Creo es mejor subirlo y dejar configurado el proyecto de codelite, ¿nó?

**Comment by adrigm** [comment]
No podemos incluir SFML dentro del proyecto porque según que SSOO y compilador necesitaremos unas librerías u otras.

Usaremos lo Wiki para explicar algunas cosas, artículos en Genbeta Dev y la documentación de la API del engine con Doxygen.

Utilcemos el tag question, para poner dudas.

**Comment by e-osuna-g** [comment]
Disculpa es mi primera vez en un proyecto, y en github así que no sabia como poner tags, acabe deduciendo que tu eras quien las ponia.

**Comment by rickyah** [comment]
Lo que si podríamos poner en el proyecto son directorios vacíos donde deberían ir las librerías externas, con un simple readme indicando cómo construirlas (o mejor un link a la wiki aquí donde lo indique)
Así al menos las rutas siempre serán las mismas para cada plataforma y podremos configurar los proyectos en base a eso.

**Comment by adrigm** [comment]
Sí, hay que hacer en la wiki un artículo de como se configura el proyecto. Quizás deberemos dejar los directorios con un txt indicativo, algo así

extlibs/libs-gcc/x86/puts-gc-x86-libs.txt

Así los directorios estarían creados y no hay duda de donde copiar las bibliotecas necesarias.

**Comment by rickyah** [comment]
Un fichero tenemos que poner fijo por que si no un directorio vacío en git no se puede agregar al repo.

¿cómo podemos organizar ese directorio para normalizar las librerías? Por ejemplo:

```
extlibs/PLATFORM/LIB_NAME/.gitignore
```

**Comment by adrigm** [comment]
He creado una serie de directorios con TXT vacíos que tienen como nombre las bibliotecas que van. ¿Podría valer así?

**Comment by rickyah** [comment]
Perfecto, mejor que el gitignore si me preguntas, así queda claro en el propio proyecto lo que hay que añadir
